wiki/People/The_Team/en.md

```diff
@@ -13,15 +13,16 @@ Active
 
 | Name | Description |
 | ---- | ----------- |
-| [Ephemeral](https://osu.ppy.sh/u/102335) | Player support, Meganekko secretary |
+| [cYsmix](https://osu.ppy.sh/u/272870) | Audio design & official osu! Featured Artist |
+| [Ephemeral](https://osu.ppy.sh/u/102335) | Media, Featured Artist outreach, Meganekko secretary |
 | [flyte](https://osu.ppy.sh/u/3103765), [osu!next](https://next.ppy.sh/) | Lead designer |
-| [Loctav](https://osu.ppy.sh/u/71366) | Official tournament organization, Staff and community management |
-| [nanaya](https://osu.ppy.sh/u/2387883) | Web implementation |
+| [nanaya](https://osu.ppy.sh/u/2387883) | osu!web developer |
+| [nekodex](https://osu.ppy.sh/u/104) | osu!web developer & official osu! Featured Artist |
 | [RBRat3](https://osu.ppy.sh/u/307202) | osu!painter, Cool stuff maker-guy extraordinaire |
 | [smoogipooo](https://osu.ppy.sh/u/1040328) | osu!mania development, Bug busting |
+| [Shiro](https://osu.ppy.sh/u/113005) | Player support |
 | [Tom94](https://osu.ppy.sh/u/1857058) | pp development |
 | [yelle](https://osu.ppy.sh/u/4916903) | [osu!store](/wiki/osu!store) management |
```

Comment:
can you remove this wiki link? (or relink to <https://osu.ppy.sh/store/listing>)